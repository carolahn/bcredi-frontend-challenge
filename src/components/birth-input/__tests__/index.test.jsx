import React from "react";
import renderer from "react-test-renderer";

import BirthInput from "../index";

describe("BirthInput render test", () => {
  it("renders correctly BirthInput", () => {
    const tree = renderer
      .create(<BirthInput values={{}} register={() => {}} errors={{}} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
