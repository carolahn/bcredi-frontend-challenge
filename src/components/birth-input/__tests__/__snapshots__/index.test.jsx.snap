// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BirthInput render test renders correctly BirthInput 1`] = `
<div
  className="register__small-input-container"
>
  <label
    className="register__label register__label--small"
  >
    Data de nascimento
  </label>
  <input
    className="sc-gsTCUz iyuuZk register__input--small"
    maxLength={10}
    name="birth"
    onChange={[Function]}
    placeholder="DD/MM/AAAA"
    style={
      Object {
        "border": "1px solid #e6e6e6",
        "color": "#333333",
      }
    }
    width="148px"
  />
</div>
`;
